src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -67,17 +58,12 @@ public CaseSearchResult searchCases(
             + "the search results will contain only metadata by default (no case field data). To get case data in the "
             + "search results, please state the alias fields to be returned in the _source property for e.g."
             + " \"_source\":[\"alias.customer\",\"alias.postcode\"]",
-                  required = true)
+            required = true)
         @RequestBody String jsonSearchRequest) {
 
         Instant start = Instant.now();
 
-        rejectBlackListedQuery(jsonSearchRequest);
-
-        CrossCaseTypeSearchRequest request = new CrossCaseTypeSearchRequest.Builder()
-            .withCaseTypes(caseTypeIds)
-            .withSearchRequest(stringToJsonNode(jsonSearchRequest))
-            .build();
+        CrossCaseTypeSearchRequest request = elasticsearchQueryHelper.prepareRequest(caseTypeIds, jsonSearchRequest);
```

Comment:
minor: I have the feeling having this class in common between the 2 endpoints internal and eternal will complicate our lives. We need the 2 endpoints controller methods to be able to evolve independently. They might have different requirements. For example we might decide to allow not passing any case type for the external searches in the future. But only for the external not the internal. If that happens, since the code is in common, you risk regressions on the internal version as well if we don't notice the code is in common.
If the goal is to remove duplication, there are better ways to do it. You can have the common utility methods in `elasticsearchQueryHelper`. But call only the ones needed from the 2 controllers